### PR TITLE
IGVF-408-add-related-donor-property2

### PR DESCRIPTION
**Pull request by khzlin**

### CODE REVIEW

File: src/igvfd/schemas/changelogs/human_donor.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 ### Minor changes since schema version 7
 
 * Add `dbxrefs`.
+* Add `related_donors`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/human_donor.md:6]: you need to bump the schema number

File: src/igvfd/schemas/human_donor.json [hunk 2]
```diff
@@ -44,12 +44,51 @@
...
+                        "title": "Related Donor Identifier",
+                        "description": "An identifier for the related donor.",
+                        "type": "string",
+                        "linkTo": "HumanDonor"
+                    },
+                    "relationship_type": {
+                        "title": "Relationship Type",
+                        "description": "A descriptive term for the related donor’s relationship to this donor.",
+                        "type": "string",
+                        "enum": [
```
Comment (gabdank) [inline src/igvfd/schemas/human_donor.json:75]: can you please add comment with URL or explanation where the relationships list are coming from?

File: src/igvfd/schemas/changelogs/rodent_donor.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 ### Minor changes since schema version 6
 
 * Add `dbxrefs`.
+* Remove `parents`.
```
Comment (gabdank) [inline src/igvfd/schemas/changelogs/rodent_donor.md:7]: need to bump the schema number (it is new version both for human and for mouse)

File: src/igvfd/schemas/human_donor.json [hunk 4]
```diff
@@ -44,12 +44,51 @@
...
+                            "aunt",
```
Comment (gabdank) [inline src/igvfd/schemas/human_donor.json:76]: should not we add uncle?

**Review by gabdank — CHANGES_REQUESTED** [review]

File: src/igvfd/schemas/human_donor.json (hunk 2 above)
Comment (khzlin) [reply src/igvfd/schemas/human_donor.json:75]: ok

File: src/igvfd/schemas/changelogs/human_donor.md (hunk 1 above)
Comment (khzlin) [reply src/igvfd/schemas/changelogs/human_donor.md:6]: ok

File: src/igvfd/schemas/changelogs/rodent_donor.md (hunk 3 above)
Comment (khzlin) [reply src/igvfd/schemas/changelogs/rodent_donor.md:7]: ok

File: src/igvfd/schemas/human_donor.json (hunk 4 above)
Comment (khzlin) [reply src/igvfd/schemas/human_donor.json:76]: ok

File: src/igvfd/schemas/human_donor.json [hunk 5]
```diff
@@ -70,7 +70,7 @@
                     },
                     "relationship_type": {
                         "title": "Relationship Type",
-                        "description": "A descriptive term for the related donor’s relationship to this donor.",
+                        "description": "A descriptive term for the related donor’s relationship to this donor. List of enums comes from http://genealogy-quest.com/glossary-terms/relationship-terms/",
```
Comment (gabdank) [inline src/igvfd/schemas/human_donor.json:73]: it should go in the comment.

**Review by gabdank — CHANGES_REQUESTED** [review]

**Review by gabdank — APPROVED** [review]

**Review by gabdank — DISMISSED** [review]

**Pull request merged**